Makefile.PL

```diff
@@ -14,6 +14,7 @@ WriteMakefile(
     VERSION_FROM  => 'lib/Test/Differences.pm',
     ABSTRACT_FROM => 'lib/Test/Differences.pm',
     PL_FILES      => {},
+    LICENSE       => 'artistic_2',
```

Comment:
" You may use, distribute and modify this software under the terms of the GNU public license, any version, *or* the Artistic license "

Not sure what that is in META.yml-ese, but it ain't just artistic_2.